### ISSUE TITLE
Remix: Named import of ColorSchemeScript throws an error

### ISSUE THREAD

**Issue by majidmadadi** [mantinedev/mantine#4787]
### What package has an issue

@mantine/core

### Describe the bug

I followed the documentation to create a remix typescript app and I came across the following error:
```

import { ColorSchemeScript, MantineProvider } from "@mantine/core";
         ^^^^^^^^^^^^^^^^^
    SyntaxError: Named export 'ColorSchemeScript' not found. The requested module '@mantine/core' is a CommonJS module, which may 
    not support all module.exports as named exports.
    CommonJS modules can always be imported via the default export, for example using:
    
    import pkg from '@mantine/core';
    const { ColorSchemeScript, MantineProvider } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at ModuleJob.run (node:internal/modules/esm/module_job:190:5)
```

### What version of @mantine/* page do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.0.0

### Do you know how to fix the issue

None

### Are you willing to participate in fixing this issue and create a pull request with the fix

None

**Comment by NikolaGrgic** [comment]
I'm experiencing the same issue using Mantine with Remix.

**Comment by KubaJastrz** [comment]
It looks like `@mantine/*` packages are incorrectly configured

https://publint.dev/@mantine/core@7.0.0
https://publint.dev/@mantine/hooks@7.0.0
https://publint.dev/@mantine/form@7.0.0
https://publint.dev/@mantine/modals@7.0.0

> ./esm/index.js is written in ESM, but is interpreted as CJS. Consider using the .mjs extension, e.g. ./esm/index.mjs ([More info](https://publint.dev/rules.html#file_invalid_format))

**Comment by cayter** [comment]
I think it would work if we're able to fix [this](https://discord.com/channels/854810300876062770/915603477806329866/1154650698886152272).

**Comment by CGuiho** [comment]
I'm facing the same problem since `mantine v7 beta` with `remix v1.19.x`!
Back with the `remix v1.19.x` the workaround was to remove the `"type": "module"` from the `package.json`.

Now in `remix v2` this workaround no longer works.

**Comment by rtivital** [comment]
The issue will be fixed in next patch, you can view the PR of the updated application here – https://github.com/mantinedev/remix-template/pull/11

**Comment by rtivital** [comment]
The issue is resolved in [7.0.1](https://github.com/mantinedev/mantine/releases/tag/7.0.1)
